### DISCUSSION TITLE
Use an organization action in a subdirectory

### DISCUSSION THREAD

**Discussion by berglie** [community/community#10264] [Actions] [answered]
Using the action in a subdirectory, such as `{owner}/{repo}/{path}@{ref}` works great on public repos. It also works from a workflow within the same action repository, which is of type internal organization repository.

However, it seems like the action can't be accessed by a different repo in the same organization. I get the following error:
`Error: An action could not be found at the URI 'https://api.github.com/repos/<org>/<repo>/zipball/xxx'`

The action is called by `{owner}/{repo}/{path}@{ref}` and is identical with the one used for testing in the action repo which works fine.

In the repository settings for actions, access for repositories in the organization is enabled in the action repo.
In both repos, both have enabled _Allow all actions_.

Are subdirectories not allowed for actions in organizations, or am I missing something?

**Comment by chrispat** [comment]
@berglie Thank you for the feedback.

I agree this should work and seems like a bug.  I will pass this along to the team so they can take a look.

**Comment by tauhid621** [comment, marked as answer]
@berglie I am not able to reproduce this issue. I can access actions from subdirectories of an internal repository. Can you please check again and share the logs. Do note that internal actions can only be accessed from internal/private repo.

**Reply by berglie** [reply]
Hi, I tested again with same code and now it works. Maybe I have to wait for some time after creating the action repository before I can use it from another repo?

In my log I get `Download action repository '<org>/<action-repo>@v1` (same as before) at job setup.
But earlier I got this message as well:
`Error: An action could not be found at the URI 'https://api.github.com/repos/<org>/<action-repo>/zipball/1291fae50ac4b25385506169c499377136babb0d'`

**Reply by tauhid621** [reply]
Actions from an internal repository feature was shipped as public beta on 20th Jan. I believe you tried to access the action before this date. Before 20th, only workflows could be used from an internal repository.

**Reply by berglie** [reply]
That makes perfect sense, thank you for your feedback
